ml-agents-envs/mlagents/envs/communicator_objects/agent_action_proto_pb2.pyi

```diff
@@ -0,0 +1,51 @@
+# @generated by generate_proto_mypy_stubs.py.  Do not edit!
```

Comment:
All of these `.pyi` files are stubs generated with https://github.com/dropbox/mypy-protobuf. I updated the directions with how to generate them.

Reply:
Why use stub files instead of putting the type annotation directly onto the python source file?

Reply:
The python source here is generated by protoc and it's not generating traditional python classes. In theory we might be able to do it, but it would involve wrestling with both `protoc` and `mypy-protobuf`.

Reply:
Oh I see, make sense.